src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidgetViewImpl.java

```diff
@@ -147,11 +147,6 @@ public void setMessage(String message) {
 	}
 
 	@Override
-	public void setActiveUsers(String activeUsers){
```

Comment:
activeUsers is only set in configure() and there is a view.clear() in configure(). This method has been removed.